openedx/features/course_experience/views/course_home.py

```diff
@@ -105,6 +105,7 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         context = {
             'csrf': csrf(request)['csrf_token'],
             'course_key': course_key,
+            'course': course,
```

Comment:
Yes, it is this commit: https://github.com/edx/edx-platform/pull/15021/commits/0a3e865b448a8edc49b15443015de3540c539646